setup.py

```diff
@@ -0,0 +1 @@
+sdk/python/setup.py
```

Comment:
Is it possible to add a comment in this file? To explain why we do this? thanks!

Reply:
It is not, this is just a standard unix symlink file unfortunately.